src/main/webapp/js/date.js

```diff
@@ -321,4 +321,25 @@ function parseDate(val) {
 		}
 	}
 	return null;
-}
\ No newline at end of file
+}
+
+function getTimeZone(date) {
+    var currentDateInString = date.toString().split(" ");
+    var userTimeZone = currentDateInString[currentDateInString.length - 1];
+    return userTimeZone;
+}
```

Comment:
Two issues here:
- `getTimeZone(...)` doesn't seem rightly used. It looks like a method that returns the time zone of a date string, but its usage is getting user's time zone (not _any_ time zone). If the usage is as such, the method should be `getUserTimeZone()` and should be independent of any parameter.
- The method's working is not so clear, so some comments might be helpful (e.g why split to string array and get the string at position of length - 1).